### ISSUE TITLE
DOM Error trying to change the FQDN for a node

### ISSUE THREAD

**Issue by Castafers** [pterodactyl/panel#4817]
### Current Behavior

![image](https://github.com/pterodactyl/panel/assets/21153318/213122ed-981f-4cdb-9298-157a639612f4)

I freshly installed Pterodactyl Panel this morning, to be presented with this issue.

Changing the FQDN of a node causes a **500 Internal Server Error**, as the changes are reverted.

The following logs are generated [here](https://pteropaste.com/z4xxh326gn15/).

Changing any of the other settings appear to be working as intended, just not the FQDN.

Supposedly from the Pterodactyl Panel Discord server, this is a PHP issue. So below are my PHP details.

```
PHP 8.1.21 (cli) (built: Jul  8 2023 07:10:19) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.21, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.21, Copyright (c), by Zend Technologies

```
```
[PHP Modules]
bcmath, calendar, Core, ctype, curl, date, dom, exif, FFI, fileinfo, filter, ftp, gd, gettext, hash, iconv, json, libxml, mbstring, mysqli, mysqlnd, openssl, pcntl, pcre, PDO, pdo_mysql, Phar, posix, readline, Reflection, session, shmop, SimpleXML, sockets, sodium, SPL, standard, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlreader, xmlwriter, xsl
Zend OPcache, zip, zlib

[Zend Modules]
Zend OPcache
```

I checked the exact file referenced in the error, and did not find corruption.
I checked my nginx configuration for the PHP arguements, and they are as intended from the official documentation [here](https://pterodactyl.io/guides/php_upgrade.html).

### Expected Behavior

![image](https://github.com/pterodactyl/panel/assets/21153318/2334d44a-663a-4cb6-8465-aa8a2a9f9970)

The domain name change is applied, the machines are rebooted, and everything works.

### Steps to Reproduce

Freshly install Pterodactyl Panel on Ubuntu 22.04.2 LTS

Follow the Pterodactyl Panel installation guide for the panel & wings.

Connect a node to the panel successfully.

Create game server(s).

Attempt to change the FQDN, and press **apply**.

### Panel Version

1.11.3

### Wings Version

1.11.6

### Error Logs

```bash
https://pteropaste.com/z4xxh326gn15
```

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by Boy132** [comment]
Duplicate of #4613

The error from the logs is outdated and was already resolved in the Discord.

**Comment by Castafers** [comment]
> Duplicate of #4613
> 
> The error from the logs is outdated and was already resolved in the Discord.

Issue #4613 states 2 different characteristics compared to mine.

- Editing "any value" causes the 500 error
- The node is offline, and wings isn't started

**A few minutes later**

I was trying to see if they were just old logs by deleting the old laravel logs. And as a result right afterwards, my node configuration changes went through without error.

I had been dealing with this issue for several hours today. I'm astonished after taking a break, and deleting the logs. That it just began working again.

**Comment by danny6167** [comment]
> > Duplicate of #4613
> > The error from the logs is outdated and was already resolved in the Discord.
> 
> Issue #4613 states 2 different characteristics compared to mine.
> 
> * Editing "any value" causes the 500 error
> * The node is offline, and wings isn't started
> 
> **A few minutes later**
> 
> I was trying to see if they were just old logs by deleting the old laravel logs. And as a result right afterwards, my node configuration changes went through without error.
> 
> I had been dealing with this issue for several hours today. I'm astonished after taking a break, and deleting the logs. That it just began working again.

Incorrect permissions on your log files is the cause of your problem.

**Comment by Castafers** [comment]
To my understanding, I believe that is the most likely cause. As Software-Noob suggests in issue #4613.

> The fact that no exceptions are written indicates there are permission issues for it to write them and is what's triggering internal errors.

Now, I'm only about a beginner when it comes to Linux. So I may or may not understand the system to a certain degree.
But I suspect that it was due to accidentally upgrading packages, then downgrading. Causing the permissions to be rewritten.
I did **reinstall** PHP from 8.2 to 8.1, the supported version. After having accidentally upgraded.
